forms/src/FieldManagerForm.test.tsx

```diff
@@ -1,78 +1,50 @@
-import { render, screen, waitFor } from '@testing-library/react'
-import { InputField } from './Input'
-import { useFieldManager } from './useFieldManager'
+import { render, screen } from '@testing-library/react'
 import userEvent from '@testing-library/user-event'
-import { FC, PropsWithChildren } from 'react'
-import { FieldManager } from './FieldManager'
+import { FieldManagerForm } from './FieldManagerForm'
+import * as useFieldManager from './useFieldManager'
 
-const spySuccess = vi.fn()
-
-const SampleFieldWithRenderError = () => {
-	const { checkFieldError, reset, fieldErrors } = useFieldManager()
-	const error = checkFieldError('field')
-	return (
-		<>
-			<InputField data-testid="field" name="field" required />
-			<button type="submit" data-testid="submit">Submit</button>
-			<button type="button" data-testid="reset" onClick={() => reset()}>Reset</button>
-			{fieldErrors.field && <div data-testid="error-state">anything</div>}
-			{error && <div data-testid="error-show">{error}</div>}
-		</>
-	)
-}
-
-const FullyControlledFieldWrapper: FC<PropsWithChildren> = ({ children }) => (
-	<FieldManager fields={{ field: '' }} onValidSubmit={spySuccess}>
-		{children}
-	</FieldManager>
-)
+vi.mock('./useFieldManager')
 
 describe('FieldManagerForm', () => {
 	beforeEach(() => {
 		vi.clearAllMocks()
 	})
 
-	it('should setup validation form and not report any errors if form is valid', async () => {
-		render(<SampleFieldWithRenderError />, { wrapper: FullyControlledFieldWrapper })
```

Comment:
These wer more like integration tests. bleh... Mocked out the field manager, tested only the behavior of the FieldManagerForm itself.